### ISSUE TITLE
Lack of visual feedback for primed lunging.

### ISSUE THREAD

**Issue by Snowofcone** [pearuhdox/Cartographer-2.0#115]
Lunging enchant only has a audio queue for when the enchant has finished charging. This could become very unclear for players who are currently being swarmed by mobs and cannot hear the queue due to the chaos. some small amount of particles could remedy this albeit it very specific particles

**Comment by pearuhdox** [comment]
Added a visual indicator for charging a Lunging or Ambushing item. This can be toggled off via /trigger toggle_charging and is on by default.
